solvcon/__init__.py

```diff
@@ -89,7 +89,7 @@
 from .cmdutil import Command, go
 from .conf import env
 from .block import Block
-import_name('Table', '.mesh', may_fail=True)
+import_name('Table', '.march', may_fail=True)
```

Comment:
I remember you mentioned the naming idea in the last sync up meeting.
We are going to use the naming "march" instead of "mesh".
Isn't it?